src/Program.cs

```diff
@@ -74,4 +75,25 @@ internal static void ConfigureServices(IServiceCollection services)
         services.AddSingleton<IPostgresService, PostgresService>();
         services.AddSingleton<CommandFactory>();
     }
+
+    internal static void LogDebugArgs(string[] args)
+    {
+        string debug = Environment.GetEnvironmentVariable("DEBUG") ?? string.Empty;
+
+        bool isDebugEnabled =
+            debug.Equals("true", StringComparison.OrdinalIgnoreCase) ||
+            debug.Contains("azure-mcp", StringComparison.OrdinalIgnoreCase) ||
+            debug == "*";
+
+        //if (isDebugEnabled)
```

Comment:
Does this compile?

Reply:
it does

Reply:
You worried about whitespace?

Reply:
nit:  do you mind yanking the commented-out if-check and removing the L89 / L97 scope